multimodal/src/autogluon/multimodal/utils/environment.py

```diff
@@ -70,8 +72,22 @@ def compute_num_gpus(config_num_gpus: Union[int, float, List], strategy: str):
     return num_gpus
 
 
+def get_state_dict_dtype(state_dict):
+    """
+    Returns the first found floating dtype in `state_dict` if there is one, otherwise returns the first dtype.
+    """
+    for t in state_dict.values():
+
+        if t.is_floating_point():
+            return t.dtype
+
+    # if no floating dtype was found return whatever the first dtype is
+    else:
```

Comment:
The `else` doesn't align with `if`.

Reply:
It is supposed to be like this. If the for loop does not return anything, it will go to this else statement.

Reply:
This is from the huggingface [sourcecode](https://github.com/huggingface/transformers/blob/67b37891335262bbbb501b0b53418aafd43a04d7/src/transformers/modeling_utils.py#L218-L228).